components/navigable-container/README.md

```diff
@@ -0,0 +1,103 @@
+NavigableContainers
+=============
+
+`NavigableContainer` is a React component to render a container navigable using the keyboard. Only things that are focusable can be navigated to. It will currently always be a `div`.
+
+`NavigableContainer` is exported as two classes: `NavigableMenu` and `TabbableContainer`. `NavigableContainer` itself is **not** exported. `NavigableMenu` and `TabbableContainer` have the props listed below. Any other props will be passed through to the `div`.
+
+----
+
+## Props
+
+These are the props that `NavigableMenu` and `TabbableContainer`. Any props which are specific to one class are labelled appropriately.
+
+### onNavigate
+
+A callback invoked when the menu navigates to one of its children passing the index and child as an argument
+
+- Type: `Function`
+- Required: No
+
+### deep
+
+A boolean to look for navigable children in the direct children or any descendant. True means that any descendant can be considered navigable, and false means only direct children are considered.
+
+- Type: `Boolean`
+- Required: No
+- default: false
+
+
+### cycle
+
+A boolean which tells the component whether or not to cycle from the end back to the beginning and vice versa.
+
+- Type: `Boolean`
+- Required: No
+- default: true
+
+## handleRef
```

Comment:
Can you clarify why we need this prop. I'd love if we can avoid it since it's too "technical".